pywbemtools/pywbemcli/_cmd_subscription.py

```diff
@@ -205,10 +205,13 @@ def subscription_add_destination(context, identity, **options):
 @click.option('--source-namespaces', type=str, metavar='TEXT',
               required=False, default=None,
               multiple=True,
-              help=u"The namespace(s) for which the query is defined. If not "
-                   u"defined, the default namespace of the server is used. "
-                   u"Define multiple namespaces by using option multiple times "
-                   u"or comma-separating namespaces names.")
+              help=u"The namespace(s) for which the query is defined. Multiple "
+                   u"values may be defined with a single comma-separated "
+                   u"string of namespaces or multiple options. If defined the "
+                   u"namespaces will be inserted into the SourceNamespaces "
+                   u"property. Otherwise the property will not be created and "
+                   u"the WBEM server should use the interop namespace for the "
+                   u"indication filter.")
```

Comment:
The sentence "the WBEM server should use ..." is a requirement on the WBEM server. However, we are on the client side here. So at best, we can state whether WBEM servers typically use the Interop namespace, or which standard requires them to use the Interop namespace.

Reply:
That is an interesting one.  It is the model that defines this :If NULL, the namespace of the Filter registration is assumed.".  I am going with typically.

Reply:
DONE